unittests/BuildSystem/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_llbuild_unittest(BuildSystemTests
+  BuildExecutionQueueTest.cpp
+  )
+set(LLBUILD_BUILDSYSTEM_UNITTEST_PATH ${CMAKE_CURRENT_LIST_DIR})
```

Comment:
@ddunbar for some reason this doesn't work but it works if set from the root CMakeList here which feels wrong to me because of hardcoded path -> https://github.com/apple/swift-llbuild/pull/27/files#diff-af3b638bc2a3e6c650974192a53c7291R66